src/components/VoterStatus.js

```diff
@@ -170,6 +173,13 @@ const VoterStatus = ({
               </a>
             </Fragment>
           )}
+          <DotSpacer />
+          {!legacy && (
+            <Fragment>
+              Total voting weight:{' '}
+              <Black>{formatRound(pollVotingPower, 4)} MKR</Black>{' '}
```

Comment:
slightly edge case-y, but -- if the user has a hot wallet connected, they don't see their connected wallet's balance in the voter status bar, but it _is_ used to calculate `pollVotingPower` so they might be surprised by the "Total voting weight" number. Again, this seems a pretty rare case. Not sure how much we want to worry about it.

Reply:
True, Chris brought it up too and said as long as it's being calculated not to worry about displaying it. We can add it pretty easily if people decide they want to see it.